universal_transfer_operator/src/universal_transfer_operator/data_providers/filesystem/base.py

```diff
@@ -107,7 +107,7 @@ def _convert_remote_file_to_byte_stream(self, file: str) -> io.IOBase:
             remote_obj_buffer.seek(0)
             return remote_obj_buffer
 
-    def write(self, source_ref):
+    def write(self, source_ref: FileStream):
```

Comment:
Please add a description for this ref in the docstring.

Reply:
Done.